zson/marshal_test.go

```diff
@@ -1,13 +1,6 @@
 package zson_test
 
-import (
-	"strings"
-	"testing"
-
-	"github.com/brimsec/zq/zson"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
+/* NOT YET
```

Comment:
Nit: Say `t.Skip("blah blah blah")` in each test rather than commenting them.

Reply:
Oops, @alfred-landrum  saw that too.  Those tests are fine and a last vestige of the code movements.  Will fix.